### PR TITLE
Follow-up for 2.0.2 release

### PR DESCRIPTION
**Pull request by kgaillot**
These changes are completions of commits from 2.0.2, which really should have been done before release. The most significant (though still pretty minor) is a fix for a 2.0.2 regression in stonith_admin --list-targets.

### CODE REVIEW

File: tools/stonith_admin.c [hunk 1]
```diff
@@ -635,53 +635,17 @@ main(int argc, char **argv)
...
-
-                            end = strchr(status, '\n');
-                            if (end != NULL) {
-                                *end = '\0';
-                            }
-                        }
-                    }
-
-                    if (hostname != NULL && uuid != NULL && status != NULL) {
-                        out->message(out, "fence-target", hostname, uuid, status);
```
Comment (clumens) [inline tools/stonith_admin.c]: This is the only place the fence-target message is used.  If you don't forsee it getting used elsewhere, we could get rid of the functions that handle it, too.

Comment (kgaillot) [reply tools/stonith_admin.c]: Oh right, I meant to and forgot.

File: doc/Makefile.am [hunk 2]
```diff
@@ -364,9 +367,6 @@ doxygen-upload: doxygen
 
 # ABI compatibility report as HTML
 
-LAST_RELEASE	?= Pacemaker-$(VERSION)
-TAG		?= $(git log --pretty=format:%H -n 1 HEAD)
-
```
Comment (jnpkrn) [inline doc/Makefile.am]: what a low-flying subtlety, good catch

File: xml/Makefile.am [hunk 3]
```diff
@@ -139,7 +140,7 @@ api/api-result-%.rng: $(API_files) Makefile.am
 	echo '      <attribute name="request"> <text /> </attribute>' >> $@
 	echo '      <optional>' >> $@
 	echo '        <choice>' >> $@
-	for rng in $(API_base); do $(top_srcdir)/xml/best-match.sh api/$$rng $(*) $(@) "          " || :; done
+	for rng in $(API_request_base); do $(top_srcdir)/xml/best-match.sh api/$$rng $(*) $(@) "          " || :; done
```
Comment (jnpkrn) [inline xml/Makefile.am]: we should eventually document that the same approaches apply
also to `api` ones, at least it seems so

**Pull request merged**